### DISCUSSION TITLE
Allow changing "notice" type by clicking icon of notice

### DISCUSSION THREAD

**Discussion by designatedsuccessor** [outline/outline#7152] [Ideas]
There are four notice types: success, warning, info, and tip.  Each type has an icon that is on the left side of the notice's message.

When in _Edit_ mode, allow clicking the icon to change the notice type.  So when you click the icon, you'd see a submenu similar to the one below:
<img width="279" alt="image" src="https://github.com/outline/outline/assets/25268183/424c8411-c68c-4e98-9129-bb7c0a43234b">

**Comment by almereyda** [comment]
Indeed, this is something that I am often missing.

Also changing a notice type when writing `/not…` and selecting one from the list in the beginning of the first line is not supported. Instead, it will remove the line from the notice box.

Maybe there would also be space for a notice formatting button in the formatting toolbar, which could alternatively be used to select another option.
